schema.js

```diff
@@ -6,6 +6,33 @@ const joi = require('joi');
 const levels = Object.values(Severity).filter(level => typeof level === 'string')
   || ['fatal', 'error', 'warning', 'log', 'info', 'debug', 'critical'];
 
+const sentryClient = joi.object().keys({
+  configureScope: joi.func().minArity(1),
+  Scope: joi.func().required(),
+  Parsers: joi.object().keys({
+    parseError: joi.func().minArity(1).required(),
+  }).unknown().required(),
+  Handlers: joi.object().keys({
+    parseRequest: joi.func().minArity(2).required(),
+  }).unknown().required(),
+  withScope: joi.func().minArity(1).required(),
+  captureEvent: joi.func().minArity(1).required(),
+}).unknown();
```

Comment:
While this is nice for now, i think this will be paint to maintain.

Does sentry expose its version somehow, so we could check against the major version in that? (and maybe just log a warning that hapi-sentry might break?)

Reply:
Unfortunately sentry does not expose something like a version. So I will leave this validation as is, which unfortunately means we have to adjust it every time we change the usage of Sentry instance methods in `index.js`.